### PR TITLE
Refactor/css styling

### PR DESCRIPTION
**Pull request by hheyhhay**
Pair Project PR Template: Rancid Tomatillos

What iteration is being worked on?
Iteration 3: Network Requests & Async JS

What is in request and being accomplished?
- CSS refactored 
- API calls refactored to be able to access details of each
- Refactored to add loading message

Collaborators: @hheyhhay @ericli1996

Other information:

PM: @sertmer @kaylagordon

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,40 +2,68 @@ import React, { Component } from 'react';
 import './App.css';
 import Movies from './Movies.js';
 
+const API = "https://rancid-tomatillos.herokuapp.com/api/v2/movies";
```
Comment (ericli1996) [inline src/App.js:5]: Great job with making this less repetitive.

File: src/App.js [hunk 2]
```diff
@@ -2,40 +2,68 @@ import React, { Component } from 'react';
...
+const DEFAULT_QUERY = ''
+
 class App extends Component {
   constructor() {
     super();
       this.state = {
         movies: [],
-        error: ''
+        error: '',
+        isLoading: false
```
Comment (ericli1996) [inline src/App.js:14]: Great job adding an isLoading key to the state

File: src/App.js [hunk 3]
```diff
@@ -2,40 +2,68 @@ import React, { Component } from 'react';
...
       }
   }
 
   componentDidMount = () => {
-    this.showAllMovies();
-  }
+    this.setState({isLoading: true})
+
+    fetch(API + DEFAULT_QUERY)
```
Comment (ericli1996) [inline src/App.js:21]: This may be repetitive

**Review by ericli1996 — APPROVED** [review]
Overall very good. The componentDidMount method seems to be fetching data twice, so double check that. Great understanding of the React Fundamentals, JS, CSS

File: src/App.js (hunk 2 above)
Comment (sertmer) [reply src/App.js:14]: isLoading probably isn't necessary - couldn't you just check to see if the array has been populated? I'm not saying you need to remove it, but you should define your reasoning if you want to keep it.

File: src/App.js (hunk 3 above)
Comment (sertmer) [reply src/App.js:21]: don't forget to define the apiCalls in it's own file, per the rubric!

**Review by sertmer — COMMENTED** [review]
Nice work! check the comments for some refactoring ideas!

**Pull request merged**